docs/source/get_started/index.rst

```diff
@@ -5,11 +5,55 @@
 First things first
 ******************
 
-With a fresh AiiDA installation, there are still a few steps to complete, before any calculations can be started:
+With ``aiida-core`` installed, a few steps remain before starting to run calculations:
 
-1) The daemon has to be configured and started
-2) A computer has to be configured
-3) A code has to configured
-4) A plugin has to be installed
+1) :ref:`plugins`
+2) :ref:`setup_computer`
+3) :ref:`setup_code`
+
+.. _plugins:
+
+***************
+Install Plugins
+***************
+
+While the ``aiida-core`` package provides the workflow engine and database model, it relies on *plugins* for connecting to specific simulation codes.
+
+Find your AiiDA plugins on the `AiiDA plugin registry <https://aiidateam.github.io/aiida-registry>`_. 
+Most plugins are also hosted on the `Python Package Index <https://pypi.org/search/?q=aiida>`_ and can be installed as follows::
+
+    pip install aiida-diff  # install 'aiida-diff' plugin from PyPI
+    reentry scan -r aiida   # notify aiida of new entry points
+
+If no PyPI package is available, you can install 
+the plugin directly from a source code repository, e.g.::
+
+    git clone https://github.com/aiidateam/aiida-diff
+    pip install aiida-diff  # install 'aiida-diff' plugin from local folder
+    reentry scan -r aiida   # notify aiida of new entry points
+
+After installing new plugins, **restart the daemon**  using ``verdi daemon restart``.
+
+.. note::
+    The reentry cache can also be updated from python when access to the commandline is not available (e.g. in jupyter notebooks).
+
+    .. code-block:: python
+
+        from reentry import manager
+        manager.scan(group_re='aiida')
+    
+.. note::
+    What does ``pip install aiida-diff`` do?
+
+    * resolves and installs the dependencies on other python packages
+    * creates a folder ``aiida_diff.egg-info/`` with metadata about the package
+    * if the ``-e`` option is given, creates a symbolic link from the python package
+      search path to the ``aiida-diff`` directory and puts the ``.egg-info``
+      folder there.
+      Changes to the **source code** will be picked up by python without reinstalling, 
+      but changes to the **metadata** will not.
```

Comment:
Maybe add another point saying that for changes will also not be picked up by running interpreters, i.e. daemon needs to be restarted and open shells need to be reopened

Reply:
thanks, will add some info concerning this in brackets.
alternatively, we could delete the entire note - it's all python /pip basics and I don't really think we should be explaining this in the aiida docs (let me know)

Reply:
I agree, maybe it would be good to add the extra detail I suggested and then move this whole section to troubleshooting

Reply:
there currently isn't a troubleshooting section for "get started".
I've left it in for the moment, let's deal with this later.

Reply:
No I meant the general troubleshooting section in the installation section, because installing plugins can be viewed as the installation part. But it is okay for now